`xcaddy build` fails with runtime error

I saw you changed the interface, so I tried updating the Homebrew formula to the new builder version. Unfortunately the build fails now:

```
$ go run cmd/xcaddy/main.go build v2.0.0-beta.20 --output /usr/local/Cellar/caddy/2.0.0-beta.20/bin/caddy
go: downloading github.com/Masterminds/semver/v3 v3.0.3
panic: runtime error: index out of range [3] with length 3

goroutine 1 [running]:
main.runBuild(0xc0000100c0, 0x3, 0x3, 0xc000016178, 0x0)
	/private/tmp/caddy--caddy-builder-20200326-4177-6b8ole/builder-1cc8e08c16745777e12ac5d7d0cd44535b0bb35e/cmd/xcaddy/main.go:72 +0x7ff
main.main()
	/private/tmp/caddy--caddy-builder-20200326-4177-6b8ole/builder-1cc8e08c16745777e12ac5d7d0cd44535b0bb35e/cmd/xcaddy/main.go:32 +0x13e
exit status 2
```

Do you know why?

--- a/cmd/xcaddy/main.go
+++ b/cmd/xcaddy/main.go
@@ -69,7 +69,7 @@ func runBuild(args []string) error {
 				return fmt.Errorf("expected value after --output flag")
 			}
 			i++
-			output = args[i+1]
+			output = args[i]
 
 		default:
 			if caddyVersion != "" {

